frontend/src/metabase/visualizations/components/LegendItem.jsx

```diff
@@ -67,7 +69,6 @@ export default class LegendItem extends Component {
         style={{
           overflowX: "hidden",
           flex: "0 1 auto",
-          opacity: isMuted ? 0.4 : 1,
```

Comment:
We actually weren't using the `muted` class from the module, so I figured we should use it in place of an inline style